platform/commonUI/general/src/controllers/ContextMenuController.js

```diff
@@ -36,12 +36,19 @@ define(
          * @memberof platform/commonUI/general
          * @constructor
          */
-        function ContextMenuController($scope) {
+        function ContextMenuController($scope, policyService) {
+
+            function applyPolicy(action) {
+                return policyService.allow('action', action,
+                    { domainObject: $scope.domainObject });
+            }
+
             // Refresh variable "menuActions" in the scope
             function updateActions() {
-                $scope.menuActions = $scope.action ?
-                        $scope.action.getActions({ category: 'contextual' }) :
-                        [];
+                var actions = $scope.action ?
+                    $scope.action.getActions({ category: 'contextual' }) : [];
+
+                $scope.menuActions = actions.filter(applyPolicy);
```

Comment:
Is this filtering necessary? Policy is enforced by an [`actionService` decorator](https://github.com/nasa/openmct/blob/d6fe543c16ba32c637b2106aaf3da07e0e49929b/platform/policy/src/PolicyActionDecorator.js#L52), so I'd expect these to already be absent.